js/client-topbar.js

```diff
@@ -242,6 +244,14 @@
 		tablist: function () {
 			app.addPopup(TabListPopup);
 		},
+		showRoomTabRightClickMenu: function (e) {
+			e.preventDefault();
+			if ($(e.currentTarget).data('chat')) {
```

Comment:
```suggestion
			if ($(e.currentTarget).data('chat')) {
				e.preventDefault();
```
Right-clicking should never do nothing.

Reply:
Ah right, that slipped my mind completely.